kubectl-rook-ceph.sh

```diff
@@ -179,6 +193,50 @@ function run_operator_restart_command () {
   kubectl --namespace "$NAMESPACE" rollout restart deploy/rook-ceph-operator
 }
 
+function path_cm_rook_ceph_operator_config() {
+  if [[ "$#" -ne 2 ]]; then
+    fail_error "require exactly 2 subcommand: $*"
+  fi
+  kubectl --namespace "$NAMESPACE" patch configmaps rook-ceph-operator-config --type json --patch  "[{ op: replace, path: /data/$1, value: $2 }]"
+}
+
+function get_mon_endpoints (){
+  end_of_command_parsing "$@" # end of command tree
+  kubectl --namespace "$NAMESPACE" get cm rook-ceph-mon-endpoints -o json | jq --monochrome-output '.data.data'| tr -d '"'
+}
+
+function rook_version () {
+  [[ -z "${1:-""}" ]] && fail_error "Missing 'version' subcommand"
+  subcommand="$1"
+  shift # remove the subcommand from the front of the arg list
+  case "$subcommand" in
+    version)
+      run_rook_version "$@"
+      ;;
+    status)
+      run_rook_cr_status "$@"
+      ;;
+    *)
+      fail_error "'operator' subcommand '$subcommand' does not exist"
+      ;;
+  esac
+}
+
+function run_rook_version () {
+  end_of_command_parsing "$@" # end of command tree
+  kubectl --namespace "$NAMESPACE" exec deploy/rook-ceph-operator -- rook version
+  kubectl --namespace "$NAMESPACE" get cephclusters.ceph.rook.io -ojson| jq --monochrome-output '.items[].status.ceph.versions' | tr -d '}' | tr -d '{'
```

Comment:
Why not just call `ceph versions` it's probably better.

Reply:
right, changed.